pygfx/utils/transform.py

```diff
@@ -93,6 +94,12 @@ def inverse_matrix(self) -> np.ndarray:
     def _decomposed(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
         return la.matrix_decompose(self.matrix)
 
+    @cached
+    def directions(self):
+        axes = np.eye(3)
+        axes[2, 2] = 1 - 2 * self.forward_is_minus_z
+        return (*la.vector_apply_matrix(axes, self.matrix),)
```

Comment:
Are you sure this aligns? I'd have expected right to be -X.

* glTF uses a right-handed coordinate system. glTF defines +Y as up, +Z as forward, and -X as right; the front of a glTF asset faces +Z.
* The camera is defined such that the local +X axis is to the right, the “lens” looks towards the local -Z axis, and the top of the camera is aligned with the local +Y axis.

Reply:
> glTF uses a right-handed coordinate system. glTF defines +Y as up, +Z as forward, and -X as right; the front of a glTF asset faces +Z.

Good catch. Positive X does indeed point left in a right-handed frame. I'll fix it.